README.md

```diff
@@ -102,3 +102,18 @@ Contributing
 =================
 
 Feel free to fork and send us pull requests. Here are the [guidelines for contribution](https://github.com/Murali-group/GraphSpace/blob/master/CONTRIBUTING.md) in GraphSpace.
+
+
+Maintainers
+=================
+
+The Project is maintained by :-
```

Comment:
Just say "If you have questions or suggestions about GraphSpace, please contact" Remove the ":-".